### ISSUE TITLE
Broken ctrl+q urls

### ISSUE THREAD

**Issue by rhazor** [PoE-TradeMacro/POE-TradeMacro#125]
E.g. if you hover over item like:
```
Rarity: Normal
Occultist's Vestment
--------
Energy Shield: 140
--------
Requirements:
Level: 62
Int: 180 (unmet)
--------
Sockets: B-B-G-R-B B 
--------
Item Level: 75
--------
6% increased Spell Damage
```
And click ctrl+q it opens an empty page with no results http://poe.trade/search/ohakisigunomau now you might think it's strange cuz no way there is no 6s 5links listed right? Cuz correct link to same settings with results is http://poe.trade/search/anaukoguharauk

**Comment by jn64** [comment]
I think this should be fixed with #119 974b6a91f45ec52045a71067af78d84e2ad6030f

It was searching for "BodyArmour" type instead of "Body Armour". If you try on any other type like a 6S 5L weapon it should fill in the form correctly.

The 2nd link you provide is incorrect as well since it's searching for *all* 6S 5L regardless of type.

**Comment by Eruyome** [comment]
Yeh that's probably the case. Update will be released soon.
